resources/views/recipes/show-all-recipes.blade.php

```diff
@@ -3,19 +3,56 @@
 @section('content')
         <h2 class="page-title">My Recipes</h2>
 
-        <nav class="recipe-mini-nav">
+        <nav class="mini-nav recipe-nav">
              <ul>
-                 <li><a><i class="fa fa-plus"></i></a></li>
+                 <li><a href="#"><i class="fa fa-plus"></i></a></li>
                  <li><a><i class="fa fa-cart-plus"></i></a></li>
                  <li><a><i class="fa fa-pencil"></i></a></li>
-                 <li><a><i class="fa fa-plus"></i></a></li>
-                 <li><a><i class="fa fa-plus"></i></a></li>
+                 <li><a><i class="fa fa-users"></i></a></li>
+                 <li><a><i class="fa fa-trash"></i></a></li>
              </ul>
         </nav>
 
-        <ul>
-        @foreach($recipes as $recipe)
-            <li><a href="recipe/{{$recipe->id}}">{{$recipe->title}}</a></li>
-        @endforeach
-        </ul>
+
+        <div class="category-wrapper">
+            <ul class="category">
+                <li class="category-title"><h3>Category Title</h3></li>
+                <li>
+                    <ul class="recipes">
+                        @foreach($recipes as $recipe)
+                            <li><a href="recipe/{{$recipe->id}}">{{$recipe->title}}</a></li>
+                        @endforeach
+                    </ul>
+                </li>
+            </ul>
+            <ul class="category">
+                <li class="category-title"><h3>Category Title</h3></li>
+                <li>
+                    <ul class="recipes">
```

Comment:
Not that you need to do this, but this is an example of where you could use the `range` function with a `foreach` loop. Essentially loop through x number of times and insert "<li>Placeholder recipe</li>"